d3a_interface/schemas.py

```diff
@@ -134,6 +134,7 @@ class ResultsSchemas:
                             "bills": {"type": "object"},
                             "tree_summary": {"type": "object"},
                             "status": {"type": "string"},
+                            "eta_seconds": {"type": "number"},
```

Comment:
One clarification, would this key value pair also contains, simulation queued time?

Reply:
No, because the simulation queued time is only determined on the fly by d3-web.